WordPress/WordPressNotificationServiceExtension/Sources/FormattableContent/RichNotificationContentFormatter.swift

```diff
@@ -21,75 +32,81 @@ class RichNotificationContentFormatter {
     init(notification: RemoteNotification, parser: FormattableContentActionParser = RemoteNotificationActionParser()) {
         self.notification = notification
         self.parser = parser
+
+        formatBody()
+        formatAttributedSubject()
     }
+}
 
-    /// Returns the body of the notification.
-    ///
-    /// - Returns: the formatted body of the notification if it exists; `nil` otherwise
-    func formatAttributedBody() -> NSAttributedString? {
-        guard
-            let body = notification.body,
+// MARK: - Private behavior
+
+private extension RichNotificationContentFormatter {
+    /// Attempts to format both a plain-text & attributed-text representation of the notification content.
+    func formatBody() {
+        guard let body = notification.body,
             let bodyBlocks = body as? [[String: AnyObject]],
-            !bodyBlocks.isEmpty
-        else
-        {
-            return nil
+            !bodyBlocks.isEmpty else {
```

Comment:
I do have some kind of "! blindness" and I find clauses starting with a negative very difficult to parse. But that is just me.

Do you think it would make sense to add a `isNotEmpty` property to Array to add some syntactic sugar and make lines like this read as `guard .... bodyBlocks.isNotEmpty`?